### PR TITLE
Use JsonFormat print records of CDC response

### PR DESCRIPTION
**Pull request by azexcy**
Changes proposed in this pull request:
  - Fix data check job not find at commit migration job
  - Use JsonFormat print records of CDC response

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by azexcy** [comment]
now the record print just like
```
{
  "before": [{
    "name": "order_id",
    "value": {
      "@type": "type.googleapis.com/google.protobuf.Empty"
    }
  }, {
    "name": "user_id",
    "value": {
      "@type": "type.googleapis.com/google.protobuf.Empty"
    }
  }],
  "after": [{
    "name": "order_id",
    "value": {
      "@type": "type.googleapis.com/google.protobuf.Int64Value",
      "value": "1"
    }
  }, {
    "name": "user_id",
    "value": {
      "@type": "type.googleapis.com/google.protobuf.Int64Value",
      "value": "39"
    }
  }]
}
```

File: kernel/data-pipeline/scenario/migration/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/api/impl/MigrationJobAPI.java [hunk 1]
```diff
@@ -423,8 +423,8 @@ private void cleanTempTableOnRollback(final String jobId) throws SQLException {
     public void commit(final String jobId) {
         log.info("Commit job {}", jobId);
         final long startTimeMillis = System.currentTimeMillis();
-        dropCheckJobs(jobId);
         stop(jobId);
+        dropCheckJobs(jobId);
```
Comment (azexcy) [inline kernel/data-pipeline/scenario/migration/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/api/impl/MigrationJobAPI.java:427]: Fix the error log from ci

```
[INFO ] 2023-03-31 09:49:18.688 [docker-java-stream-786712462] :ShardingSphere-Proxy - STDOUT: [WARN ] 2023-03-31 09:49:18.163 [Connection-3-ThreadExecutor] o.a.s.d.p.s.m.a.impl.MigrationJobAPI - stop related check job failed, check job id: j0202p0000j0102p0000af4a124b05e9bb3c7ce635279b74d91d1, error: Can not find pipeline job `j0202p0000j0102p0000af4a124b05e9bb3c7ce635279b74d91d1`.
[INFO ] 2023-03-31 09:49:18.688 [docker-java-stream-786712462] :ShardingSphere-Proxy - STDOUT: [INFO ] 2023-03-31 09:49:18.186 [Pipeline-EventListener-1] o.a.s.d.p.c.m.n.e.h.i.ConfigMetaDataChangedEventHandler - UPDATED job configuration: /pipeline/jobs/j0102p0000af4a124b05e9bb3c7ce635279b74d91d/config, disabled=true
[INFO ] 2023-03-31 09:49:18.688 [docker-java-stream-786712462] :ShardingSphere-Proxy - STDOUT: [INFO ] 2023-03-31 09:49:18.186 [Pipeline-EventListener-1] o.a.s.d.p.c.job.AbstractPipelineJob - stop tasks runner, jobId=j0102p0000af4a124b05e9bb3c7ce635279b74d91d
```

File: kernel/data-pipeline/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java [hunk 2]
```diff
@@ -36,7 +44,18 @@ public static void main(final String[] args) {
...
         // and mysql-connector-java 5.x version will ignore serverTimezone jdbc parameter and use the default time zone in the program
         // TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
-        StartCDCClientParameter parameter = new StartCDCClientParameter(records -> log.info("records: {}", records));
+        TypeRegistry registry = TypeRegistry.newBuilder().add(EmptyProto.getDescriptor().getMessageTypes()).add(TimestampProto.getDescriptor().getMessageTypes())
+                .add(WrappersProto.getDescriptor().getMessageTypes()).build();
+        Printer printer = JsonFormat.printer().usingTypeRegistry(registry);
+        StartCDCClientParameter parameter = new StartCDCClientParameter(records -> {
+            for (Record each : records) {
+                try {
+                    log.info("record: {}", printer.print(each));
```
Comment (sandynz) [inline kernel/data-pipeline/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java:53]: Could we remove `@type`? (e.g. `"@type": "type.googleapis.com/google.protobuf.Empty"`)

**Review by sandynz — CHANGES_REQUESTED** [review]

File: kernel/data-pipeline/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java (hunk 2 above)
Comment (azexcy) [reply kernel/data-pipeline/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java:53]: image
seem it's necessary for JSON representation of an Any value

https://protobuf.dev/reference/protobuf/google.protobuf/

**Comment by azexcy** [comment]
Its just for print log and will think mor, so close it now.

**Pull request closed without merging**